docs/resources/sql_table.md

```diff
@@ -82,11 +82,12 @@ The following arguments are supported:
 * `properties` - (Optional) Extensible Table properties.
 
 ### `column` configuration block
+
 For table columns
 Currently, changing the column definitions for a table will require dropping and re-creating the table
 
 * `name` - User-visible name of column
-* `type` - Column type spec (with metadata) as SQL text
+* `type` - (Optional for `VIEW`) Column type spec (with metadata) as SQL text
```

Comment:
Is it optional, or is it invalid to specify with view?

Reply:
good point, I have updated the doc